ppsci/arch/base.py

```diff
@@ -80,10 +81,26 @@ def split_to_dict(
         data = paddle.split(data_tensor, len(keys), axis=axis)
         return {key: data[i] for i, key in enumerate(keys)}
 
-    def register_input_transform(self, transform):
+    def register_input_transform(
+        self,
+        transform: Callable[[Dict[str, paddle.Tensor]], Dict[str, paddle.Tensor]],
+    ):
+        """Register input transform.
+
+        Args:
+            transform (Callable[[Dict[str, paddle.Tensor]], Dict[str, paddle.Tensor]]): Transform, parameter and return of it should be one dict.
+        """
         self._input_transform = transform
 
-    def register_output_transform(self, transform):
+    def register_output_transform(
+        self,
+        transform: Callable[[Dict[str, paddle.Tensor]], Dict[str, paddle.Tensor]],
+    ):
+        """Register output transform.
+
+        Args:
+            transform (Callable[[Dict[str, paddle.Tensor]], Dict[str, paddle.Tensor]]): Transform, parameter and return of it should be one dict.
```

Comment:
Output transform of network, receive a single tensor dict and return a single tensor dict.